test/runnable/imports/bar10378.d

```diff
@@ -0,0 +1,4 @@
+
+module bar;
```

Comment:
That's actually a feature of D. The file name doesn't have to match the module name.